<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*{
color: #78799c;
border: none;
}

#firstWidget, #frame_2, #frame_5{
	background-color: #2a2c49;
}

#mainBody,#Form{
	background-color: #1a1f39;
}

QLineEdit{
	background-color:transparent;
}

#searchFrame, QLineEdit{
	border-radius: 10px;
	border: 2px solid #78799c;
}

#appHeader{
	color:#78799c;
}

#prodottiHeader{
	color:  #a06ce3;
}

#avantiarw{
	border: 1px solid #a06ce3;
	background-color: #a06ce3 ;
	border-radius: 5px;
	
}

#finestreSecondarie{
	background-color: #2a2c49;
	border: solid;
	border-radius: 20px;
}

QLabel{
	color:#78799c;
}

#mainFrame{
	background-color: #2a2c49;
}

#homeBtn{
	color:#78799c;
	background-color:#1a1f39;
	padding : 10px 5px;
	text-align:left;
	border-top-left-radius:25px;
}
QPushButton{
	padding : 10px 5px;
	text-align:left;
	color:#78799c;
	font: 10pt &quot;Leelawadee UI&quot;;
	background-color:#1a1f39;
    border: solid;
    border-radius: 11px;
}


#frame_3{
	background-color:#2a2c49;
	border: solid;
	border-radius: 20px;
}
QTableWidget{
	background-color:#2a2c49;
	border: solid;
}
#leftMenu,#frame{
background-color: #2a2c49;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameInVendita">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titoloInVendita">
        <property name="text">
         <string>Prodotti In Vendita</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tabInVendita"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameScaduti">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titoloScaduti">
        <property name="text">
         <string>Prodotti Scaduti</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tabScaduti"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameVenduti">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titoloVenduti">
        <property name="text">
         <string>Prodotti Venduti</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tabVenduti"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
